cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -31,7 +31,7 @@ set(config_module_list
 	drivers/ll40ls
 	drivers/trone
 	drivers/gps
-	drivers/pwm_out_sim
+	#drivers/pwm_out_sim
```

Comment:
Right, thanks for the hint. I removed `sdlog2` instead, I didn't see a better candidate.